clients/python/girder_client/__init__.py

```diff
@@ -140,16 +151,20 @@ def __init__(self, host=None, port=None, apiRoot=None, scheme=None,
         :param cacheSettings: Settings to use with the diskcache library, or
             None to disable caching.
         """
-        if apiUrl is None:
+        configApiUrl = config.get('girder_client', 'apiUrl', fallback=None)
```

Comment:
We should use the config file as a fallback if the options aren't passed to the constructor rather than having the config file take precedence.